### DISCUSSION TITLE
Field hooks for lexical blocks feature didnt work.

### DISCUSSION THREAD

**Discussion by dudubs** [payloadcms/payload#4351] [Feature Requests & Ideas]
### Describe the Bug

When I write a hooks for text or any else type (afterChange, beforeRead and all other hook types).

### To Reproduce

Code for example:

`
 BlocksFeature({
        blocks: [
          {
            slug: "video",
            fields: [
              {
                name: "url",
                type: "text",

                hooks: {
                  afterRead: [
                    () => {
                      console.log("Not called");
                    },
                  ],
                  beforeChange: [
                    () => {
                      console.log("Not called");
                    },
                  ],
                  afterChange: [
                    () => {
                      console.log("Not called");
                    },
                  ],
                },
              },
            ],
          },
        ],
      })

`

### Payload Version

2.2.2

### Adapters and Plugins

editor-lexical

**Comment by AlessioGr** [comment]
Hey, I've converted it into a feature request. Lexical blocks indeed do not support hooks yet, though that will be added some time in the future

**Reply by BrianJM** [reply]
@AlessioGr How does this fit into prioritization with v3 in alpha/beta? I was under the impression that Lexical (in v2) would be a bit more developed/stable before v3, but I also understand the importance of v3.

**Reply by maxmorozoff** [reply]
@BrianJM I've briefly checked the alpha branch, and it seems that it's not yet implemented.

https://github.com/payloadcms/payload/blob/0f7106bf4aad7e1841a8ea65a4c72844a7ca040b/packages/richtext-lexical/src/populate/recurseNestedFields.ts#L62
